extension/src/setup/index.ts

```diff
@@ -378,9 +378,18 @@ export class Setup
     const commandArgs = args.length === 0 ? '' : ` ${args.join(' ')}`
     const command = executable + commandArgs
 
+    if (!cwd) {
```

Comment:
I believe that the test coverage is at 66% is due to this if statement not getting covered. Should I add an additional test or do I need to rewrite something?